sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/CloseReason.cs

```diff
@@ -21,7 +21,7 @@ public enum PartitionProcessorCloseReason
         /// <summary>A non-retriable exception was thrown by the Event Hub Client.</summary>
```

Comment:
Not sure this is entirely accurate.   Any exception that exhausts retries (or was never retriable) from any point in the library code can trigger this.    I'd rename to just "Error" or "Exception" or similar.

Reply:
I agree. I'll make `EventHubException` and `ProcessEventException` a single `CloseReason.Exception` for now, but I'll need to revisit this when working on error handling.